src/utils/ParameterConstraintsUtils.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+const operatorsDict = {
+  date: ['>', '<', '>=', '<=', '==', '!='],
+  number: ['>', '<', '>=', '<=', '==', '!='],
+  int: ['>', '<', '>=', '<=', '==', '!='],
+  string: ['==', '!='],
+  bool: ['==', '!='],
+};
+
+const getVarTypesComparison = (varType, varTypeToCompare) => {
+  const numberVarTypes = ['int', 'number'];
+  return varType === varTypeToCompare || [varType, varTypeToCompare].every((el) => numberVarTypes.includes(el));
+};
+
+const getParameterValidationConstraint = (validationString, varType, parametersList) => {
+  if (validationString) {
+    const [operator, id] = validationString.replace(/\s+/g, ' ').trim().split(' ');
```

Comment:
Maybe we should return `null` early if `id` is `undefined` at this point? (when the constraint string has no "space" character inside it)

I fear that proceeding to the lines below with an `undefined` id may lead to edge cases and bugs